crates/lakekeeper/src/server/tables/load_table.rs

```diff
@@ -59,6 +79,18 @@ pub(super) async fn load_table<C: CatalogStore, A: Authorizer + Clone, S: Secret
     )
     .await?;
 
+    let etag = get_etag(&table_info);
+    if etag_already_present(
+        &etags,
+        etag.clone()
+            .map(|e| e.trim_matches('"').to_string())
+            .as_ref(),
```

Comment:
Nit: We have two useless clones in here: `.clone()` and `to_string()`.

```suggestion
        etag.as_ref().map(|e| e.trim_matches('"'))
```

This won't compile because `etag_already_present` requires `&String` instead of `&str` for its second argument. We should almost always prefer `&str` to `&String`.